hw/ip/otbn/util/docs/get_impl.py

```diff
@@ -2,10 +2,11 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
+from dataclasses import dataclass
```

Comment:
Dataclasses were added in Python 3.7. You can add a polyfill from PyPi for Python 3.6, however people will then need to update their Python dependencies again to build the docs. Maybe for this small use case it would be best to avoid them.

Reply:
Oh, good point. This was to match the code in libcst, but I agree it isn't really needed here. I'll drop it.

Reply:
Ah ok, libcst already depends on the dataclasses polyfill, so users should already have that: https://github.com/Instagram/LibCST/blob/master/requirements.txt

You can add the same line to our python-requirements.txt and we're fine.